crates/ironrdp-rdpdr/src/pdu/efs.rs

```diff
@@ -1589,3 +1597,250 @@ bitflags! {
         const FILE_OVERWRITTEN = 0x00000003;
     }
 }
+
+/// [2.2.3.3.8 Server Drive Query Information Request (DR_DRIVE_QUERY_INFORMATION_REQ)]
+///
+/// Note that Length, Padding, and QueryBuffer fields are all ignored in keeping with the [analogous code in FreeRDP].
+///
+/// [2.2.3.3.8 Server Drive Query Information Request (DR_DRIVE_QUERY_INFORMATION_REQ)]: https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-rdpefs/e43dcd68-2980-40a9-9238-344b6cf94946
```

Comment:
Nit: you can ignore if you want, but I was thinking about only putting a link on the section number instead:
```suggestion
/// [2.2.3.3.8] Server Drive Query Information Request (DR_DRIVE_QUERY_INFORMATION_REQ)
///
/// Note that Length, Padding, and QueryBuffer fields are all ignored in keeping with the [analogous code in FreeRDP].
///
/// [2.2.3.3.8]: https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-rdpefs/e43dcd68-2980-40a9-9238-344b6cf94946
```
This requires less duplication and results in shorter lines. What do you think?

Reply:
I'm fine with that style in general, but I'd be bothered by the inconsistency with the rest of [this part of] the codebase if I were to fix just this one. Seems more trouble than it's worth to fix them all at this point.

Reply:
Fear not! [`sd`](https://github.com/chmln/sd) is here to save the day!
I’ll run that in another PR though, you’re right.

Reply:
https://github.com/Devolutions/IronRDP/pull/247